src/Arcus.Messaging.Tests.Workers.EventHubs.Core/Arcus.Messaging.Tests.Workers.EventHubs.Core.csproj

```diff
@@ -11,6 +11,7 @@
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.0" />
     <PackageReference Include="Microsoft.Extensions.Hosting" Version="6.0.0" />
     <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.9.5" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.8.4" />
```

Comment:
Isn't Microsoft.SqlClient the new package ?

Reply:
Ah, not sure. Was using this package in our Web API W3C stuff, too.

Reply:
I think we should use the Microsoft.Data.SqlClient package
https://learn.microsoft.com/en-us/sql/connect/ado-net/introduction-microsoft-data-sqlclient-namespace?view=sql-server-ver16

Reply:
Yes, will change this.

Reply:
Updated in https://github.com/arcus-azure/arcus.messaging/pull/344/commits/7ae36db2b1f43d6a0d256633c1018544bb3c4a0e